discordrb.json

```diff
@@ -57,6 +57,18 @@
             "voiceReceive": "no",
             "voiceSend": "yes",
             "voiceStateUpdate": "yes"
+        },
+        "misc": {
+            "supportsUserLogin": "yes",
```

Comment:
Well, they do need such features in order to have a complete library. Even though it's deprecated it's not explicitly disallowed and people still use it.